sdk-scheme-adapter/outbound/v2_0_0/components/schemas/individualTransfer.yaml

```diff
@@ -2,29 +2,24 @@ title: IndividualTransfer
 type: object
 description: Data model for the complex type 'individualTransfer'.
```

Comment:
This seems like its a standard FSPIOP Bulk definition....

Should we not rather just move this definition into the FSPIOP spec or even better reference it there if it exists?

Reply:
Should I just move this file to https://github.com/mojaloop/api-snippets/blob/master/fspiop/v1_1/openapi3/components/schemas/IndividualTransfer.yaml (with overwrite)?
They have some differences, like `ilpPacket` and `condition` param. Not sure if we use spec file from fspiop dir anywhere.

Reply:
Well spotted...I didn't notice that difference, so I hereby retract my statement.